src/nwm_routing/src/nwm_routing/__main__.py

```diff
@@ -495,7 +495,7 @@ def main_v02(argv):
         if verbose:
             print("creating usgs time_slice data array ...")
 
-        usgs_df = nnu.build_data_assimilation(data_assimilation_parameters)
+        usgs_df, _ = nnu.build_data_assimilation(data_assimilation_parameters)
```

Comment:
This function returns 2 variables. Without this change `usgs_df` is a tuple and creates downstream havoc.